recipes/zserio/all/conanfile.py

```diff
@@ -50,13 +63,21 @@ def validate(self):
         if self.settings.compiler.cppstd:
             check_min_cppstd(self, self._min_cppstd)
 
+        minimum_compiler_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_compiler_version and Version(self.settings.compiler.version) < minimum_compiler_version:
+            raise ConanInvalidConfiguration(
+                f"Compiler version '{self.settings.compiler.version}' not supported, "
+                f"minumum is '{minimum_compiler_version}'!"
+            )
+
     def source(self):
         sources = self.conan_data["sources"][self.version]
         get(self, **sources["runtime"], strip_root=True)
         download(self, filename="LICENSE", **sources["license"])
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.variables["CMAKE_CXX_STANDARD"] = self._min_cppstd
```

Comment:
```suggestion
        tc.variables["CMAKE_CXX_STANDARD"] = self.settings.compiler.get_safe("cppstd", str(self._min_cppstd)).replace("gnu", "")
```
It will respect settings.compiler.cppstd in case defined, otherwise, will define 11 (min_cppstd)